### ISSUE TITLE
no-unused-class for scss mixins

### ISSUE THREAD

**Issue by uny** [atfzl/eslint-plugin-css-modules#69]
_mixins.scss:
```scss
@mixin for-mobile-screen {
  @media screen and (max-width: 768px) {
    & {
      @content;
    }
  }
}
```

index.module.scss:
```scss
@use "mixins";

.xxx {
  @include mixins.for-mobile-screen {
    // ...
  }
}
```

index.tsx
```typescript
import styles from "./index.module.scss";
```

Then, eslint says
```
error  Unused classes found in index.module.scss: for-mobile-screen  css-modules/no-unused-class
```

`mixins.for-mobile-screen` is not a selector, so this warning is false.
How can I fix this?

**Comment by uny** [comment]
Sorry I forgot that this package is not maintained.

**Comment by connorjsmith** [comment]
For anyone that encounters this, if you import the mixin without a namespace, you won't encounter the error:

```scss
@use "mixins" as *;

@include for-mobile-screen {
  /* ... */
}
```
